fabric-object-builders-v0/build.gradle

```diff
@@ -1,7 +1,8 @@
 archivesBaseName = "fabric-object-builders"
-version = getSubprojectVersion(project, "0.1.3")
+version = getSubprojectVersion(project, "0.1.4")
 
 dependencies {
     compile project(path: ':fabric-api-base', configuration: 'dev')
     compile project(path: ':fabric-mining-levels-v0', configuration: 'dev')
+	compile project(path: ':fabric-object-builders-v1', configuration: 'dev')
```

Comment:
![](https://user-images.githubusercontent.com/6596629/76778035-d9d97180-67b1-11ea-9831-a022d2f883c4.png)

Could you change the two lines above to use tabs as well?

Reply:
Yea that's an obvious one, lemme get a feed and I'll fix it